x11-libs/cairo/cairo-1.14.12.recipe

```diff
@@ -0,0 +1,106 @@
+System (via both Xlib and XCB), quartz, win32, and image buffers, \
+as well as PDF, PostScript, and SVG file output. Experimental backends \
+include OpenGL, BeOS, OS/2, and DirectFB."
+HOMEPAGE="http://cairographics.org/"
+COPYRIGHT="2000, 2002, 2004-2007 Keith Packard
+	2002-2003 University of Southern California
+	2004-2010 Red Hat, Inc.
+	2005-2010 Mozilla Corporation
+	2006-2009 Adrian Johnson
+	2007-2009 Chris Wilson
+	2006-2013 Intel Corporation
+	2011 Andrea Canciani
+	2011 Samsung Electronics
+	2010-2011 Linaro Limited
+	2009-2010 Eric Anholt
+	2002-2010 many others"
+LICENSE="GNU LGPL v2.1
+	MPL v1.1"
+REVISION="1"
+SOURCE_URI="http://cairographics.org/releases/cairo-$portVersion.tar.xz"
+CHECKSUM_SHA256="8c90f00c500b2299c0a323dd9beead2a00353752b2092ead558139bd67f7bf16"
+PATCHES="cairo-$portVersion.patchset"
+
+ARCHITECTURES="!x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="?x86"
+
+libVersion="2.11400.12"
+libVersionCompat="$libVersion compat >= ${libVersion%%.*}"
+
+PROVIDES="
+	cairo$secondaryArchSuffix = $portVersion compat >= 1
+	lib:libcairo$secondaryArchSuffix = $libVersionCompat
+	lib:libcairo_script_interpreter$secondaryArchSuffix = $libVersionCompat
+	lib:libcairo_gobject$secondaryArchSuffix = $libVersionCompat
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libfontconfig$secondaryArchSuffix
+	lib:libfreetype$secondaryArchSuffix
+	lib:libglib_2.0$secondaryArchSuffix
+	lib:libintl$secondaryArchSuffix
+	lib:libpixman_1$secondaryArchSuffix
+	lib:libpng16$secondaryArchSuffix
+	lib:libz$secondaryArchSuffix
+	"
+
+PROVIDES_devel="
+	cairo${secondaryArchSuffix}_devel = $portVersion compat >= 1
+	devel:libcairo$secondaryArchSuffix = $libVersionCompat
+	devel:libcairo_script_interpreter$secondaryArchSuffix = $libVersionCompat
+	devel:libcairo_gobject$secondaryArchSuffix = $libVersionCompat
+	"
+REQUIRES_devel="
+	cairo$secondaryArchSuffix == $portVersion base
+	devel:libfontconfig$secondaryArchSuffix
+	devel:libpixman_1$secondaryArchSuffix
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	devel:libfontconfig$secondaryArchSuffix
+	devel:libfreetype$secondaryArchSuffix
+	devel:libglib_2.0$secondaryArchSuffix
+	devel:libpixman_1$secondaryArchSuffix
+	devel:libpng16$secondaryArchSuffix
+	devel:libz$secondaryArchSuffix
+	"
+BUILD_PREREQUIRES="
+	cmd:aclocal
+	cmd:autoconf
+	cmd:automake
+	cmd:gcc$secondaryArchSuffix
+	cmd:ld$secondaryArchSuffix
+	cmd:libtoolize
+	cmd:make
+	cmd:pkg_config$secondaryArchSuffix
+	"
+
+defineDebugInfoPackage cairo$secondaryArchSuffix \
+	$libDir/libcairo.so.$libVersion \
+	$libDir/libcairo-script-interpreter.so.$libVersion \
+	$libDir/libcairo-gobject.so.$libVersion
+
+BUILD()
+{
+	autoreconf -vfi
+	export CFLAGS=-D__BSD_VISIBLE
+	runConfigure ./configure
+	make $jobArgs
+}
+
+INSTALL()
+{
+	make install
+
+	rm $libDir/libcairo*.la
+
+	prepareInstalledDevelLibs libcairo libcairo-script-interpreter libcairo-gobject
+	fixPkgconfig
+
+	packageEntries devel \
+		$developDir
```

Comment:
Well, this is a bit special, because we're not in `$docDir` but in `$dataDir`, so we can't move it to `$developDocDir`. (And there is no `developDataDir`.)